assets/js/base/context/providers/cart-checkout/payment-methods/actions.ts

```diff
@@ -2,106 +2,77 @@
  * Internal dependencies
  */
 import { ACTION, STATUS } from './constants';
-import type { PaymentMethods } from './types';
+import type { PaymentMethods, ExpressPaymentMethods } from './types';
 
 export interface ActionType {
 	type: ACTION | STATUS;
 	errorMessage?: string;
 	paymentMethodData?: Record< string, unknown >;
-	paymentMethods?: PaymentMethods;
+	paymentMethods?: PaymentMethods | ExpressPaymentMethods;
 	shouldSavePaymentMethod?: boolean;
 }
 
 /**
- * Used to dispatch a status update only for the given type.
+ * All the actions that can be dispatched for payment methods.
  */
-export const statusOnly = ( type: STATUS ): { type: STATUS } => ( { type } );
+export const actions = {
```

Comment:
I refactored this a bit to match checkoutState (return an actions object rather than individual functions). It just makes it slightly clearer when consuming from context.